nuget/lib/dependabot/nuget/nuget_config_credential_helpers.rb

```diff
@@ -48,6 +50,36 @@ def self.add_credentials_to_nuget_config(credentials)
         File.write(user_nuget_config_path, nuget_config)
       end
 
+      sig { params(credentials: T::Array[Dependabot::Credential]).returns(T::Hash[String, String]) }
+      def self.get_credentials_to_credential_helper_env(credentials)
+        env = {}
+
+        nuget_credentials = credentials.select { |cred| cred["type"] == "nuget_feed" }
```

Comment:
This feature only works with NuGet feeds from Azure DevOps, right?  If that's the case it would be a good idea to filter `credentials` to only those feeds so you don't accidentally expose feeds/tokens that aren't ADO to the ADO service.

If I recall, all Azure DevOps NuGet feeds look like `https://pkgs.dev.azure.com/...` or `https://SOME-ORGANIZATION.pkgs.visualstudio.com/...`.